gui/default/syncthing/device/globalChangesModalView.html

```diff
@@ -17,8 +17,16 @@
           <tr ng-repeat="changeEvent in globalChangeEvents">
             <td ng-if="changeEvent.data.modifiedBy">{{friendlyNameFromShort(changeEvent.data.modifiedBy)}}</td>
             <td ng-if="!changeEvent.data.modifiedBy"><span translate>Unknown</span></td>
-            <td>{{changeEvent.data.action}}</td>
-            <td>{{changeEvent.data.type}}</td>
+            <td ng-switch="changeEvent.data.action">
```

Comment:
Wouldn't it be easier to treat these as keys directly?  Just like we do for the `theme-name-*` strings. Like this:

    <td>{{("change-action-" + changeEvent.data.action) | translate}}</td>

That would allow some more context to distinguish from other uses of the single words.

Reply:
I've given this a try, but for me, it seems that a clean and non-hard-coded solution without fiddling around with JS could be to allow syntax like the one described at https://toptal.com/angular-js/internationalize-your-angularjs-app, which is

```
<span translate="change-action-modified">modified</span>
<span translate="change-action-deleted">deleted</span>
```

and so on. The current `translate.go` script doesn't support it though. I've tried to incorporate the syntax into the script, but my Go skills and knowledge are extremely rudimentary and I've only managed to go as far as save the value to a variable when it's non empty. Not really sure where exactly it should be passed then and added to json as name though.

Reply:
Well it's not impossible to add strings to the translation template without help from the script. Apparently it does catch the theme names, so maybe have a look at how that's done (I don't remember now).

Reply:
The theme names are actually a combination of Go, JS, and the actual filenames, so it's quite different than this one. Here, I only want to let the script accept `<div translate="key">value</div>` syntax universally :wink:.

I've now given this a try and modified the script code to use `translate` attribute value as key name when present. Please have a look and let me know if this is the right way to go.

Also, do you think that the two other strings, i.e. `file` and `folder` should also follow the same `change-type-*` pattern, or is it OK to use them as is?